docs/install.rst

```diff
@@ -0,0 +1,210 @@
+.. inclusion-marker-start-do-not-remove
+
+Installation Guide
```

Comment:
Consider renaming to "Horovod Installation Guide" for SEO

Reply:
Does that affect the title in the navigation bar at https://horovod.readthedocs.io/en/latest/api.html ?

Reply:
Yes

Reply:
Okay, I suppose that's okay.  Doesn't look out of place.